packages/autocomplete-js/src/elements/Input.ts

```diff
@@ -37,18 +35,7 @@ export const Input: AutocompleteElement<InputProps, HTMLInputElement> = ({
     ...autocompleteScopeApi,
   });
 
-  setProperties(element, {
-    ...inputProps,
-    onKeyDown(event: KeyboardEvent) {
-      if (onDetachedEscape && event.key === 'Escape') {
-        event.preventDefault();
-        onDetachedEscape();
-        return;
-      }
```

Comment:
This was no longer needed because the logic in `onDetachedEscape` has been covered by [this](https://github.com/algolia/autocomplete/blob/8fa038b914a1b76270a106f5fe2b223aa657d6ae/packages/autocomplete-js/src/autocomplete.ts#L239) since https://github.com/algolia/autocomplete/pull/556.

This is likely stale code from the first [Autocomplete Touch (now Detached Mode) PR](https://github.com/algolia/autocomplete/pull/379).